src/main/java/ch/baloise/corellia/api/entities/Cancellation.java

```diff
@@ -26,13 +22,9 @@
   @JsonPropertyDescription("A code uniquely identifying the cancellation reason")
   private Integer reasonCode;
```

Comment:
We had a talk with Busy and they now need the action type too. I would suggest to reuse the `Action ` type and drop the `effectiveDate` and use the one in `Action`. The UGVS service will be changed by @schlijo

Reply:
Changed :+1: